src/pages/storage/StorageList.tsx

```diff
@@ -130,29 +137,38 @@ const StorageList: FC = () => {
             />
           )}
           {!isLoading && !hasStoragePools && (
-            <EmptyState
-              iconName="pods"
-              iconClass="empty-storage-icon"
-              title="No storage found"
-              message="There are no storage pools in this project."
-            >
-              <>
-                <p>
-                  <a
-                    href="https://linuxcontainers.org/lxd/docs/latest/explanation/storage/"
-                    target="_blank"
-                    rel="noreferrer"
-                  >
-                    Learn more about storage
-                    <Icon className="external-link-icon" name="external-link" />
-                  </a>
-                </p>
-                <AddStorageBtn
-                  project={project}
-                  className="empty-state-button"
-                />
-              </>
-            </EmptyState>
+            <Row className="empty-state">
+              <Col size={6} className="col-start-large-4">
+                <EmptyState
+                  image={
+                    <Icon
+                      name="pods"
+                      className="empty-state-icon empty-storage-icon"
```

Comment:
We want to give up this approach because it creates a lot of CSS classes, and just for the purpose of slightly changing the colour of the icon. You can achieve the same with [opacity](https://github.com/canonical/anbox-cloud-dashboard/blob/main/templates/sass/styles.scss#L282) and that allows you to have a single shared CSS class for all the icons. Please check how this is done in Anbox dashboard.

Reply:
No, never use inline styles. In the storybook we use them because we are forced to, and also to make the code examples self-contained. We want to have a single CSS class that can be used for all the empty state icons throughout the app. Just like we do in Anbox.